composer/workflows/integrated_example.py

```diff
@@ -0,0 +1,140 @@
+
+DEFAULT_ARGS = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': YESTERDAY,
+    # If you'd like to test out enabling e-mail notifications on DAG
+    # retries/failure, configure your environment to support email by
+    # setting the SENDGRID variables and modifying the email fields below.
+    'email': ['email@foobar.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    'dataflow_default_options': {
+        'project': Variable.get('gcp_project'),
+        'stagingLocation': '{0}/{1}'.format(BUCKET, 'staging/'),
+        'tempLocation': '{0}/{1}'.format(BUCKET, 'temp/'),
+    }
+}
+
+with DAG('integrated_example', default_args=DEFAULT_ARGS) as dag:
+  # We need to create a BigQuery dataset to store the output from Dataflow.
+  # There's currently no BigQuery dataset creation operator to do this, so
+  # we use a `BashOperator` catch-all operator. Notice that the
+  # `create_bq_dataset_if_not_exist` command is implemented using the
+  # same command that we'd use in the BigQuery command line tools.
+  #
+  # After this completes, a BigQuery dataset with the `BQ_DATASET_NAME`
+  # should be viewable from your BigQuery dashboard.
+  create_bq_dataset_if_not_exist = """
+    bq ls {0}
+    if [ $? -ne 0 ]; then
+      bq mk {0}
+    fi
+  """.format(BQ_DATASET_NAME)
+  CREATE_BIGQUERY_DATASET = BashOperator(
+      task_id='create-bq-dataset',
+      bash_command=create_bq_dataset_if_not_exist,
+      dag=dag)
+
+  # This DataFlow operator runs a Python pipeline from Apache Beam,
+  # `hourly_team_score.py`, that calculates the scores of teams in batch and
+  # sends the results to our BigQuery dataset.
+  #
+  # Optional: We use this Apache pipeline file directly by setting the py_file as
+  # follows; however, if you'd like to see how the Dataflow oeprator uses local
+  # Python pipeline files, copy the pipeline file (https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/complete/game/hourly_team_score.py)
+  # to a directory in your bucket (we use `pipelines/`):
+  #   gsutil hourly_team_score.py gs://{composer-bucket}/pipelines
+  # Then, modify the `py_file` argument in the given DAG to point to the local
+  # file location:
+  #   py_file=`/home/airflow/gcs/pipelines/hourly_team_score`
+  #
+  # Notice how we feed in the data file `game_data-condensed.csv` (located in a
+  # public bucket `gs://example-datasets) and point the pipeline to our BigQuery
+  # dataset. As this runs (should take 3-5 minutes), you should be able to track
+  # the progress in your Dataflow dashboard as a job with the prefix
+  # `run-dataflow-python`.
+  RUN_DATAFLOW_PYTHON = DataFlowPythonOperator(
+      task_id='run-dataflow-python',
+      py_file='apache_beam.examples.complete.game.hourly_team_score',
+      py_options=['-m'],
+      options={
+          'input': 'gs://example-datasets/game_data_condensed.csv',
```

Comment:
Do you know who owns the `example-datasets` bucket? Please reach out over email/chat with this info. If this is Googler-generated data we can host at `cloud-samples-data` where it is backed up and has redundant billing ownership.